src/styles/molecules/_molecules.card.scss

```diff
@@ -22,7 +14,7 @@ $card-content-padding:   var(--spacer) !default;
  */
 
 .m-card__header {
-  padding: $card-header-padding;
+  padding: var(--spacer-xs);
```

Comment:
Willen we deze niet overschrijfbaar maken (bovenaan bedoel ik dan)?

Reply:
De meeste CSS properties die overschrijfbaar zijn, zijn kleuren. Ik zag niet meteen de use case dat een afnemer de padding van een card wil overschrijven. Daarom dat ik die weg had gedaan. 😅